de_web_plugin.cpp

```diff
@@ -1292,14 +1287,7 @@ void DeRestPluginPrivate::apsdeDataIndication(const deCONZ::ApsDataIndication &i
 
         case VENDOR_CLUSTER_ID:
         // case DE_CLUSTER_ID:
-            if (zclFrame.manufacturerCode() == VENDOR_PHILIPS)
-            {
-                handlePhilipsClusterIndication(ind, zclFrame, device);
-            }
-            else // Shouldn't we check for DE manufacturer code?
-            {
-                handleDEClusterIndication(ind, zclFrame);
-            }
+            handleDEClusterIndication(ind, zclFrame);
```

Comment:
Feel free to remove the whole `handleDEClusterIndication()` function as it is basically a NOP wasting cycles :)

Reply:
Done